drivers/hmis/app/graphql/types/hmis_schema/service_input.rb

```diff
@@ -22,8 +22,10 @@ class HmisSchema::ServiceInput < Types::BaseInputObject
     def to_params
       result = to_h
 
+      result[:record_type] = record_type || type_provided&.split(':')&.first&.to_i
       result[:type_provided] = type_provided.split(':').last&.to_i if type_provided.present?
       result[:sub_type_provided] = sub_type_provided.split(':').last&.to_i if sub_type_provided.present?
+      result[:date_provided] = date_provided || Date.today
```

Comment:
Date provided should be a required input from the user
```suggestion
```

Reply:
fixed